modal/image.py

```diff
@@ -856,6 +856,28 @@ def build_dockerfile(version: ImageBuilderVersion) -> DockerfileSpec:
             force_build=self.force_build or force_build,
         )
 
+    def entrypoint(
+        self,
+        *entrypoint_files: Union[str, List[str]],
```

Comment:
tiny nit: maybe `entrypoint_args` might be a clearer name since they form a command according to the [docs](https://docs.docker.com/reference/dockerfile/#entrypoint)